Comp2023/src/main/java/frc/robot/Constants.java

```diff
@@ -142,13 +138,28 @@
       ELBOW_UP            // Move elbow up
     }
 
+    public enum ElbowAngle
```

Comment:
The elbow joint will have a ELBOW_LOAD mode as well (until we figure out if it's the same).

Reply:
fixed